dashbuilder/dashbuilder-shared/dashbuilder-js/pom.xml

```diff
@@ -31,6 +31,13 @@
   <name>Dashbuilder JS</name>
   <description>Dashbuilder JS APIs and internal components for displaying data</description>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>appformer-js</artifactId>
```

Comment:
What about `provided`?
In addition, if we want to make sure none of these modules (appformer-js, appformer-js-monaco and dashbuilder-js) are compiled in parallel here it is better to depend on `appformer-js-monaco` so the the flow will be hardcoded as `appformer-js` -> `appformer-js-monaco` -> `dashbuilder-js`
Wdyt?

Reply:
Agree!